CMakeLists.txt

```diff
@@ -43,6 +43,10 @@ include(GNUInstallDirs)
 find_package(DD4hep 1.21 REQUIRED COMPONENTS DDCore DDRec)
 find_package(fmt REQUIRED)
 
+# ePIC compile definitions
+option(EPIC_COMPILE_DEFINITIONS "Additional compile definitions" "")
+add_compile_definitions(${EPIC_COMPILE_DEFINITIONS})
```

Comment:
cmake already respects CXXFLAGS from the environment variable. Any reason to not use that?

Reply:
No real functional reason other than a deeply engrained reflex towards compiler abstraction :-)

`CXXFLAGS` is passed to the compiler unchanged, while `add_compile_definitions()` does 'the right thing' for whichever compiler is active (not as relevant here since I suspect that LLVM uses the same flag for this).

Reply:
I'll make sure this works with `CXXFLAGS` and revert to using that.

Reply:
Reverted that commit since we can use both env var `CXXFLAGS` and cmake var `CMAKE_CXX_FLAGS`.

Reply:
Reminder: `-DCMAKE_CXX_FLAGS` is a bit clunky as it does not influence `CMAKE_CXX_FLAGS_RELEASE` and such.